tests/pytests/unit/modules/test_groupadd.py

```diff
@@ -0,0 +1,283 @@
+import pytest
+
+import salt.modules.groupadd as groupadd
+from tests.support.mock import MagicMock, patch
+
+try:
+    import grp
+except ImportError:
+    pass
+
+
+pytestmark = [
+    pytest.mark.skip_on_windows,
+]
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {groupadd: {}}
+
+
+def test_add():
```

Comment:
Can we also add a test that ensures if the command does not exist that we raise the `CommandExecutionError`